md_api/src/main/java/com/wipro/ats/bdre/md/api/ArrangePositions.java

```diff
@@ -48,17 +42,17 @@ public ArrangePositions() {
     @Autowired
     private ProcessDAO processDAO;
 
+    public ArrangePositions() {
+        AutowireCapableBeanFactory acbFactory = getAutowireCapableBeanFactory();
+        acbFactory.autowireBean(this);
+    }
+
     public Map<String, PositionsInfo> getListPositionInfo(Integer processId) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) The Cyclomatic Complexity of this method "getListPositionInfo" is 21 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)